### DISCUSSION TITLE
How to power save

### DISCUSSION THREAD

**Discussion by cheeckhicks** [prasanthrangan/hyprdots#1286] [Q&A] [answered]
It looks all very nice, but it slows down my system and sucks up all the battery. Gaming mode is good but the system still uses too much battery. What are the things that uses power and battery at most? Does translucency effect power a lot? Where do I configure those settings?

**Comment by kRHYME7** [comment, marked as answer]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/055282ea-2c7f-4728-82e2-eb79e5a6327a)


we got some presets here try to use it first. 

The one that consumes a lot is the border angle animations
